pages/common/sc_analysis_dump.md

```diff
@@ -0,0 +1,8 @@
+# sc_analysis_dump
+
+> Dump of traceroute data in a format that is easily parsed.
+> More information: <https://www.caida.org/catalog/software/scamper/>.
+
+- Output the traceroute of warts files one after the other in an easy to parse format:
```

Comment:
```suggestion
> Dump of `traceroute` data in an easily parsed format.
> More information: <https://www.caida.org/catalog/software/scamper/>.

- Output the `traceroute` of `warts` files one after the other in an easy-to-parse format:
```

Reply:
If you write traceroute in commas, you might think that this refers to the `traceroute` program, but here the subroutine `trace` of scamper is meant. Do you think I should change this anyway or could this lead to confusion?